lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -344,7 +345,9 @@ sub ia_json :Chained('ia_base') :PathPart('json') :Args(0) {
                       url => 'https://github.com/'.$pull_request{author}
                   );
 
-                  my $value = to_json \%dev_hash;
+                  my @dev_array = [\%dev_hash];
+
+                  my $value = to_json \@dev_array;
```

Comment:
What about `my $value = to_json [\%dev_hash]` since this is the only place we use `@dev_array` it seems. Might be cleaner.